provisioning/device/devdoc/polling_state_machine_requirements.md

```diff
@@ -17,11 +17,13 @@ Register round-trips one step of the registration process, not returning until a
 
 **SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_012: [** `register` shall call `PollingTransport.registrationRequest`. **]**
 
+**SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_06_001: [** If `PollingTransport.registrationRequest` succeeds with status==registering, `register` shall wait, then attempt `PollingTransport.registrationRequest` again. **]**
```

Comment:
>registering [](start = 124, length = 11)

really? what's the difference between registering and assigning? this is a weird status ...

Reply:
We use registering to re-submit a register request.  Assigning is used to resubmit a query operation.  So it's not weird.